### PR TITLE
Departments API / Page / Seeds

### PR DESCRIPTION
**Pull request by ryanlabouve**

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // learn more about it in the docs: https://pris.ly/d/prisma-schema
 
 generator client {
-  provider = "prisma-client-js"
```
Comment (ryanlabouve) [inline prisma/schema.prisma:5]: Sorry for the white-space diff here

File: prisma/schema.prisma [hunk 2]
```diff
@@ -562,52 +561,65 @@ model EmailAddressEmployee {
...
+
+model Department {
+  id        String   @id @default(uuid()) @db.Uuid
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  departmentCode String @unique
+  departmentName String
+
+  // Does each account need their own version of this list?
```
Comment (ryanlabouve) [inline prisma/schema.prisma:622]: @colinfrederickson @bigcountrycrane does each org need a copy of their own departments? Or is this fine to share app-wide?

Comment (colinfrederickson) [reply prisma/schema.prisma:622]: fine to share app-wide at the moment!

**Review by johnmosesman — APPROVED** [review]
lgtm

**Pull request merged**